OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -57,6 +57,7 @@ public static ScrollItemWidget Setup(ScrollItemWidget template, Func<bool> isSel
 			w.IsVisible = () => true;
 			w.IsSelected = isSelected;
 			w.OnClick = onClick;
+			w.ItemKey = Convert.ToString(Game.CosmeticRandom.Next());
```

Comment:
Why are you converting the key to a string? In any case, we shouldn't just be assigning random numbers as keys here.